sci-biology/finchtv/finchtv-1.3.1-r3.ebuild

```diff
@@ -25,12 +24,10 @@ src_install() {
 	exeinto /opt/bin
 	doexe finchtv
 
-	dodoc ReleaseNotes.txt
+	dodoc -r SampleData
+	docompress -x /usr/share/doc/${PF}/SampleData
```

Comment:
personally, I would install `examples` instead, given that's a semi-standardised location

Reply:
Done